app/components/map/clusters/index.js

```diff
@@ -14,27 +12,40 @@ class Clusters extends PureComponent {
     return (
       <View>
         {this.props.markers.map((marker, index) => {
+          const { datasetSlug } = this.props;
           const coordinates = {
             latitude: marker.geometry.coordinates[1],
             longitude: marker.geometry.coordinates[0]
           };
-          return marker.properties.point_count !== undefined ? (
-            <ClusterMarker id={`cluster-marker-${index}`} key={`cluster-marker-${index}`} marker={marker} zoomTo={this.props.zoomTo} datasetSlug={this.props.datasetSlug} />
-          ) : (
+
+          if (marker.properties.point_count !== undefined) {
+            return (
+              <ClusterMarker
+                id={`cluster-marker-${index}`}
+                key={`cluster-marker-${index}`}
+                marker={marker}
+                zoomTo={this.props.zoomTo}
+                datasetSlug={this.props.datasetSlug}
+              />
+            );
+          }
+          let markerColor = styles.gladColor;
+          const id = `${marker.geometry.coordinates[0]}${marker.geometry.coordinates[1]}`;
+          if (this.props.reportedAlerts.includes(id)) {
+            markerColor = styles.reportedColor;
+          } else if (datasetSlug === 'viirs') {
```

Comment:
I don't get this. If not reported then viirs? shouldn't we check first for datasetSlug and afterwards for reported or not?